pyensembl/download_cache.py

```diff
@@ -111,8 +112,12 @@ def __init__(
                 annotation_name=annotation_name,
                 annotation_version=annotation_version)
 
-            self._cache_directory_path = datacache.get_data_dir(
-                subdir=self.cache_subdirectory)
+            if environ.get(ENV_KEY) is None:
```

Comment:
Isn't the `if` handled by `datacache`?

https://github.com/hammerlab/datacache/blob/master/datacache/common.py#L29-L33